int32 scan1_index                   # Index of the scan 1
int32 scan2_index                   # Index of the scan 2
float32 yaw_diff_rad                # yaw differnce between scan1 and scan2 according to scanContext (0 if N/A)